gtk4/Gir.toml

```diff
@@ -561,6 +560,15 @@ name = "Gtk.BuilderListItemFactory"
 status = "generate"
 trust_return_value_nullability = false
 
+[[object]]
+name = "Gtk.CallbackAction"
+status = "generate"
+    [[object.function]]
+    name = "new"
+    [[object.function.parameter]]
+    name = "callback"
+    nullable = false
```

Comment:
Why? It's not marked as nullable here

Reply:
I.e. did someone break annotations after 4.0.3 or why is this currently necessary here?

Reply:
Otherwise you have to pass in `Option<Box<F>>`, I don't see why you would create a `CallbackAction` that does nothing?

Reply:
Sure but in my .gir the callback is not marked as `nullable` so why is it necessary to override this here?

Reply:
The CallbackAction takes a `GtkShortcutFunc` defined as such `
pub type GtkShortcutFunc = 
    Option<unsafe extern "C" fn(*mut GtkWidget, *mut glib::GVariant, gpointer) -> gboolean>;`, like all other typed callbacks. I suppose maybe is this reason why its end up with a nullable parameter?

Reply:
Seems like a gir bug then :)

Reply:
Opened an issue at https://github.com/gtk-rs/gir/issues/1104

Reply:
IIUC, the callback parameter is marked `nullable = "1"` in `Gtk-4.0.gir`. See line 10 305.